lib/minitest/reporters/progress_reporter.rb

```diff
@@ -38,6 +38,12 @@ def start
         show
       end
 
+      def before_test(test)
+        super
+        puts
```

Comment:
This extra `puts` seems to "break" lines with the default progressbar reporter :/